api/src/main/java/org/apache/iceberg/Snapshot.java

```diff
@@ -162,6 +162,25 @@ default Iterable<DeleteFile> removedDeleteFiles(FileIO io) {
    */
   String manifestListLocation();
 
+  /**
+   * Return the size of this snapshot's manifest list file. Must be a verified value, taken from a
```

Comment:
Confused here, we have a default of -1 as well set in base Snapshot which seemed to also be allowed as an "unset". Should we mention that here or is it always required?